files/build_templates/sonic_debian_extension.j2

```diff
@@ -116,6 +116,14 @@ sudo cp {{swsssdk_py2_wheel_path}} $FILESYSTEM_ROOT/$SWSSSDK_PY2_WHEEL_NAME
 sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip install $SWSSSDK_PY2_WHEEL_NAME
 sudo rm -rf $FILESYSTEM_ROOT/$SWSSSDK_PY2_WHEEL_NAME
 
+# Install SwSS SDK Python 3 package
+if [ -e {{swsssdk_py3_wheel_path}} ]; then
```

Comment:
why is this needed?

Reply:
If the packages requesting swsssdk_py3 whl are not included in the build, the file may not exist. It is a safety check.

Reply:
if the path is not there, then we should fail the build because we need to install the pacakge.

Reply:
Let us assume ZTP package is the only one that requires py3. If ENABLE_ZTP is not set, the py3 package is not created. But that doesn't mean the build should fail.